src/resources/nfts/simplehash/index.ts

```diff
@@ -113,3 +114,48 @@ export async function fetchSimpleHashNFTListing(
   );
   return cheapestListing;
 }
+
+/**
+ * Given an NFT, refresh its contract metadata on SimpleHash. If we can't,
+ * refresh metadata for this NFT only.
+ * @param nft
+ */
+export async function refreshNFTContractMetadata(nft: UniqueAsset) {
+  const chain = nft.isPoap
+    ? SimpleHashChain.Gnosis
+    : getSimpleHashChainFromNetwork(nft.network);
+
+  if (!chain) {
+    throw new Error(
+      `refreshNFTContractMetadata: no SimpleHash chain for network: ${nft.network}`
+    );
+  }
+
+  try {
+    await nftApi.post(
+      `/nfts/refresh/${chain}/${nft.asset_contract.address}`,
+      {},
+      {
+        headers: {
+          'Accept': 'application/json',
+          'Content-Type': 'application/json',
+          'x-api-key': NFT_API_KEY,
+        },
+      }
+    );
+  } catch {
+    // If the collection has > 20k NFTs, the above request will fail.
+    // In that case, we need to refresh the given NFT individually.
+    await nftApi.post(
```

Comment:
Also need to handle this thrown error.

And we should log what these errors are too. Probs as warnings, because we're expecting these to maybe occur sometimes. But we should keep track of how often it happens, and if it ever throws an UNexpected error that will help us keep track of those too.